project_national/fact_sheets_category_5.yml

```diff
@@ -0,0 +1,593 @@
+        costs: &costs_ashp_ducted_SEER_20_11_HSPF
+        # Source: Custom regression by Brennan Less. Ducted heat pump project costs were regressed on nameplate tons and HSPF,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. January 2019 $ inflated to April 2023 $ using 1.21 factor.
+        # Ducted ASHP regression n=317. Implicitly includes electrical upgrades and electric backup.
+          - value: 10229.58 # for HSPF 11
+            multiplier: Fixed (1)
+          - value: 167.64
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: *lifetime_ducted_ASHP_elec_backup
+
+      - *HP_cooling_efficiency_ducted
+      - *100_percent_conditioned
+      - *cooling_setpoint_no_offset
+      - *cooling_setpoint_offset_magnitude_0
+      - *cooling_setpoint_offset_period_none
+      - *heating_setpoint_no_offset
+      - *heating_setpoint_offset_magnitude_0
+      - *heating_setpoint_offset_period_none
+
+  - upgrade_name: Higher efficiency ducted ASHP w/ elec backup - existing natural gas 
+    # Measure 3
+    package_apply_logic: *ducted_ASHP_elec_backup_existing_NG
+    options:
+      #ASHP
+      - *ducted_ASHP_HigherEfficiency_elec_backup
+
+      - *HP_cooling_efficiency_ducted
+      - *100_percent_conditioned
+      - *cooling_setpoint_no_offset
+      - *cooling_setpoint_offset_magnitude_0
+      - *cooling_setpoint_offset_period_none
+      - *heating_setpoint_no_offset
+      - *heating_setpoint_offset_magnitude_0
+      - *heating_setpoint_offset_period_none
+
+
+  - upgrade_name: Higher efficiency ducted ASHP w/ elec backup - existing propane 
+    # Measure 3
+    package_apply_logic: *ducted_ASHP_elec_backup_existing_propane
+    options:
+      #ASHP
+      - *ducted_ASHP_HigherEfficiency_elec_backup
+
+      - *HP_cooling_efficiency_ducted
+      - *100_percent_conditioned
+      - *cooling_setpoint_no_offset
+      - *cooling_setpoint_offset_magnitude_0
+      - *cooling_setpoint_offset_period_none
+      - *heating_setpoint_no_offset
+      - *heating_setpoint_offset_magnitude_0
+      - *heating_setpoint_offset_period_none
+
+  - upgrade_name: Higher efficiency ducted ASHP w/ elec backup - existing fuel oil 
+    # Measure 3
+    package_apply_logic: *ducted_ASHP_elec_backup_existing_FO
+    options:
+      #ASHP
+      - *ducted_ASHP_HigherEfficiency_elec_backup
+
+      - *HP_cooling_efficiency_ducted
+      - *100_percent_conditioned
+      - *cooling_setpoint_no_offset
+      - *cooling_setpoint_offset_magnitude_0
+      - *cooling_setpoint_offset_period_none
+      - *heating_setpoint_no_offset
+      - *heating_setpoint_offset_magnitude_0
+      - *heating_setpoint_offset_period_none
+
+
+  - upgrade_name: EnergyStar ductless MSHP w/ elec backup - existing elec 
+    # Measure 4
+    package_apply_logic: &ductless_MSHP_elec_backup_existing_elec
+      - HVAC Has Ducts|No
+      - Heating Fuel|Electricity
+      - or: &not_MSHP_greater_than_SEER_16_9-2_HSPF
```

Comment:
There is one MSHP that wouldn't get upgraded, but it is so rare in our baseline that it's not in the 30k test buildstock. We need to be careful there.